docs/core/deploying/deploy-with-cli.md

```diff
@@ -1,201 +1,154 @@
+author: thraka
+ms.author: adegeo
+ms.date: 01/16/2019
 dev_langs: 
   - "csharp"
   - "vb"
 ms.custom: seodec18
 ---
-# Deploy .NET Core apps with command-line interface (CLI) tools
+# Publish .NET Core apps with the CLI
 
-You can deploy a .NET Core application either as a *framework-dependent deployment*, which includes your application binaries but depends on the presence of .NET Core on the target system, or as a *self-contained deployment*, which includes both your application and the .NET Core binaries. For an overview, see [.NET Core Application Deployment](index.md).
+This article demonstrates how you can publish your .NET Core application from the command line. .NET Core provides three ways to publish your applications. *Framework-dependent deployment* produces a .dll file that uses the locally installed .NET Core runtime. *Framework-dependent executable* produces a platform-specific executable that uses the locally installed .NET Core runtime. *Self-contained* produces a platform-specific executable and includes a local copy of the .NET Core runtime.
 
-The following sections show how to use [.NET Core command-line interface tools](../tools/index.md) to create the following kinds of deployments:
+For an overview of these deployment modes, see [.NET Core Application Deployment](index.md). 
 
-- Framework-dependent deployment
-- Framework-dependent deployment with third-party dependencies
-- Self-contained deployment
-- Self-contained deployment with third-party dependencies
+Looking for some quick help on using the CLI? The following table shows some examples of how to publish your app.
 
-When working from the command line, you can use a program editor of your choice. If your program editor is [Visual Studio Code](https://code.visualstudio.com), you can open a command console inside your Visual Studio Code environment by selecting **View** > **Integrated Terminal**.
+| Publish Mode | SDK Version | Command |
+| ------------ | ----------- | ------- |
+| Framework-dependent deployment | 2.1 | `dotnet publish -f netcoreapp2.1` |
+|                                | 2.2 | `dotnet publish -f netcoreapp2.2` |
+|                                | 3.0 | `dotnet publish -f netcoreapp3.0` |
+|                                | 2.x* | `dotnet publish` |
+| Framework-dependent executable | 2.2 | `dotnet publish -f netcoreapp2.2 -r <RID> --self-contained false` |
+|                                | 3.0 | `dotnet publish -f netcoreapp3.0 -r <RID> --self-contained false` |
+|                                | 3.0* | `dotnet publish` |
+| Self-contained deployment      | 2.1 | `dotnet publish -f netcoreapp2.1 -r <RID> --self-contained true` |
+|                                | 2.2 | `dotnet publish -f netcoreapp2.2 -r <RID> --self-contained true` |
+|                                | 3.0 | `dotnet publish -f netcoreapp3.0 -r <RID> --self-contained true` |
 
```

Comment:
I'll think about how to do that. I can't get too complex because the point of this table is we get users asking just "WHATS THE COMMAND LINE?"  

Technically I could just leave out the default modes and put in there the absolute CLI command, this would lessen the confusion?

What do you think of that?